v0.6.0 is missing code

Hi, I tried this:

```bash
cd
mkdir ohno
cd ohno
npm i @intlify/vue-i18n-loader
node -e 'require("@intlify/vue-i18n-loader")'
```

This was the result:

```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '@intlify/vue-i18n-loader'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:122:20)
    at Object.runInThisContext (vm.js:329:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at evalScript (internal/bootstrap/node.js:590:27)
```

This makes sense when you look at the list of files, and notice it doesn't include any code:

```
$ ls node_modules/@intlify/vue-i18n-loader
CHANGELOG.md  LICENSE  package.json  README.md
```

To make matters worse, v0.6.0 is the only version published on npm, and trying to install by using something like this in your package.json will also fail:

```
"@intlify/vue-i18n-loader": "intlify/vue-i18n-loader"
```

Instead, if you're like me and want to install this package, you'll need to put this in your package.json:

```
"@intlify/vue-i18n-loader": "intlify/vue-i18n-loader#v0.5.0"
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,5 +1,6 @@
 coverage
 node_modules
+lib
 .DS_Store
 *.log
 *.swp

--- a/.npmignore
+++ b/.npmignore
@@ -2,7 +2,6 @@
 *.log
 *.swp
 *.yml
-lib
 coverage
 __snapshots__
 test

